### DISCUSSION TITLE
Separate pods for RR web and temporal activites

### DISCUSSION THREAD

**Discussion by graid2030** [roadrunner-server/roadrunner#1407] [Q&A] [answered]
1. I want to use temporal activities in a separate container from the web, while still preserving the ability for the web version of the application to connect to temporal. The temporal num_workers option of 0 is not helpful for me because it means launching worker instances based on the number of cores. Am I correct in understanding that for the web version, I can ignore the temporal configuration section and just use the TEMPORAL_CLI_ADDRESS environment variable to specify the server address?
2. Is it also true that for one pod, it is recommended to use only a combination of one RR application with a PHP web worker and one RR with a Temporal activity worker?
3. Is there a community for discussing RoadRunner / Temporal on Telegram or Slack?

Thank you!

**Comment by rustatian** [comment, marked as answer]
Hey @graid2030 👋🏻 
1. You can register any number of activities and call them from any place by their name, using the `task-queue` and namespace. I'm not sure what you mean by `web` version.
2. You have control only over the workers, responsible for handling `activities`. Internally, we also have 1 workflow PHP worker, and if you use more than 1 PHP [worker](https://github.com/temporalio/samples-php/blob/master/app/worker.php#L26) with a different task-queue, we will also add one more temporal worker.
3. 
Samples: https://github.com/temporalio/samples-php/
Our discord: https://discord.gg/TFeEmCs
Docs: https://docs.temporal.io/application-development?lang=php
Temporal slack: https://t.mp/slack
Temporal forum: https://community.temporal.io/
